src/Workspaces/Remote/ServiceHub/Services/SymbolSearchUpdate/RemoteSymbolSearchUpdateService.cs

```diff
@@ -65,14 +42,14 @@ public ValueTask<ImmutableArray<PackageWithTypeResult>> FindPackagesWithTypeAsyn
 
         public ValueTask<ImmutableArray<PackageWithAssemblyResult>> FindPackagesWithAssemblyAsync(string source, string assemblyName, CancellationToken cancellationToken)
         {
-            return RunServiceAsync(cancallationToken =>
+            return RunServiceAsync(cancellationToken =>
                 _updateEngine.FindPackagesWithAssemblyAsync(source, assemblyName, cancellationToken),
```

Comment:
🤦‍♂️

Reply:
Found by teh spell checker.  I sent them feedback and kudos how this helped us!